File: src/cs/exceptions.cs
class Program
{
	static void DoStuff(bool inError)
	{
		if (inError)
        {
            throw new System.Exception("An error occured!");
        }
        System.Console.WriteLine("Doing stuff!");
	}
	
	public static void Main()
	{
		try
        {
            DoStuff(false);
            DoStuff(true);
            DoStuff(false);
        }
        catch (System.Exception ex)
        {
            System.Console.WriteLine($"ERROR: {ex.Message}");
        }
	}
}

File: src/vb/exceptions.vb
Public Module Program
    Sub DoStuff(ByVal inError as Boolean)
        If inError
            Throw New System.Exception("An error occured")
        End If
        System.Console.WriteLine("Doing stuff!")
    End Sub

    Sub Main()
        Try
            DoStuff(False)
            DoStuff(True)
            DoStuff(False)
        Catch ex As System.Exception
            System.Console.WriteLine("ERROR: " & ex.Message)
        End Try
    End Sub
End Module